webcompat/static/js/lib/bugform.js

```diff
@@ -485,7 +492,7 @@ function BugForm() {
       // grab the data URI from background-image property, since it's already
       // in the DOM: 'url("data:image/....")'
       this.previewEl.get(0).style.backgroundImage.slice(5, -2),
-      function(response) {
+      _.bind(function(response) {
```

Comment:
I think there was miscommunication, I remember asking in the review about why a _.bind() was used but I was not asking to remove it, just asking for clarification. when I was taught, "this" and "bind()" it was done in a really confusing way so I still have to think about it in usage.

in the current master version, I'm getting an error like this when trying to save an issue:
```
TypeError: this.addImageURL is not a function bugform.js:491:9
```